Classes/Twig/Typo3Loader.php

```diff
@@ -82,7 +88,14 @@ private function findTemplate(string $name, bool $throw = false): string
             throw new \Twig_Error_Loader($this->errorCache[$name]);
         }
 
-        $path = GeneralUtility::getFileAbsFileName($name);
+        $templatePath = array_values($this->cache)[0];
```

Comment:
Furthermore you cannot "include" or "reference" other templates relative to the current template. The template names are passed to the loaded without any context. The easiest way is to add a `\Twig_Loader_Filesystem` to the chain and add the path the template files are located in.